code/clue.js

```diff
@@ -27,38 +145,158 @@ const suspects = [
   profPlum,
   missScarlet,
   mrsPeacock,
-  mrMustard
-]
+  colonelMustard
+];
+console.log(suspects);
 
-const weapons = []
 
-const rooms = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
+console.log(weapons);
+
+
+const rooms = [
+  "LivningRoom",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "BillardRoom",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "LivingRoom",
+  "Observatory",
+  "Theatre",
+  "GuestHouse",
+  "Patio"
+];
+
+console.log(room);
+
+const times = [
+  "afternoon",
+  "midnight",
+  "morning",
+  "evening",
+  "dawn",
+  "dusk"
+];
+
+console.log(times);
+
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FOR THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
+
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
-}
+};
+
+console.log(randomSelector(suspects).firstName + "" + (suspects).lastName);
+console.log(randomSelector(weapons).name);
+console.log(randomSelector(room));
```

Comment:
If you check to console you actually get an error as `undefined` here, which means that this lines can't be read

Reply:
I understand.